src/modules/blockchain/implementation/polygon/polygon-service.js

```diff
@@ -7,6 +7,10 @@ class PolygonService extends Web3Service {
         this.baseTokenTicker = 'MATIC';
         this.tracTicker = 'mTRAC';
     }
+
+    getBlockchainId() {
+        return 'polygon';
```

Comment:
Let's read blockchain id from configuration, since we could have multiple implementations (set of contracts deployed) on same blockchain

Reply:
Done